helmfile.d/0320.nginx-ingress.yaml

```diff
@@ -60,6 +60,17 @@ releases:
           enabled: {{ env "NGINX_INGRESS_METRICS_ENABLED" | default "false" }}
         metrics:
           enabled: {{ env "NGINX_INGRESS_METRICS_ENABLED" | default "false" }}
+      ### Optional: NGINX_INGRESS_TCP; e.g. 8080:default/example-tcp-svc:9000,8081:staging/example-tcp-svc:9000
+      tcp:
+      {{- range env "NGINX_INGRESS_TCP" | splitList "," }}
+        {{ regexReplaceAll "(\\d*):(.*)" . "$1: $2" }}
```

Comment:
can you explain in comments what the regex does?

Reply:
We need to map 
`8080:default/example-tcp-svc:9000,8081:staging/example-tcp-svc:9000`
to
```
8080: default/example-tcp-svc:9000
8081: staging/example-tcp-svc:9000
```

Reply:
can you describe the regex in a simple # comment?
what it does and why.